web/assets/css/build.less

```diff
@@ -46,12 +46,16 @@
   }
 }
 
+.timestamped-logs {
+  white-space: pre-line;
+}
+
 .timestamped-line {
   display: flex;
   flex-direction: row;
 
   .timestamped-content {
-    word-break: break-all;
+    word-break: normal;
```

Comment:
Only breaking on word boundaries isn't the best - I tried applying these styles to https://ci.concourse-ci.org/builds/2782173, and you end up with a really long horizontal scroll:

<img width="1280" alt="Screen Shot 2020-09-22 at 9 28 02 AM" src="https://user-images.githubusercontent.com/26583442/93888392-f224ce80-fcb5-11ea-9a8a-efb3b11f7d02.png">

Reply:
Let me take a quick second look at this!